tests/deriver/conftest.py

```
@@ -0,0 +1,401 @@
+    with patch("src.deriver.queue.os.getenv") as mock_getenv:
+        mock_getenv.return_value = "2"  # 2 workers for testing
+        manager = QueueManager()
+        yield manager
+
+
+@pytest_asyncio.fixture
+async def embedding_store(sample_data):
+    """Provides a CollectionEmbeddingStore for testing."""
+    test_app, test_user = sample_data
+    collection_id = str(uuid4())
+    store = CollectionEmbeddingStore(test_app.public_id, test_user.public_id, collection_id)
```

Pylint (3.3.7)

[error] No value for argument 'collection_id' in constructor call

(E1120)